R/rsad.R

```diff
@@ -1,5 +1,6 @@
-rsad <- function(S,frac,sad, Pois.samp=TRUE,k,zeroes=FALSE,ssize=1, ...){
+rsad <- function(S, frac, sad=c("gamma", "geom", "lnorm","rnbinom", "rpoilog", "rweibull"), Pois.samp=TRUE, k, zeroes=FALSE, ssize=1, ...){
```

Comment:
Why not 
rsad <- function(S, frac, sad=c("gamma", "geom", "lnorm", "nbinom", "poilog", "weibull")
?